int32 motorIndex
---
bool isMotorReady
